### PR TITLE
Combine PDOX and PDOX.Germany

### LINKED ISSUE #7330: Ability to have normal and advanced chronosphere
Steps to reproduce:
- Be allies, france or england. Enemy is germany
- Build a chronosphere
- Capture the german construction yard
- Build an advanced chronosphere (normal chronosphere has vanished from the build menu)

### PR DESCRIPTION
**Pull request by Amelia55555**
Fixes #7330

### CODE REVIEW

File: mods/ra/rules/structures.yaml [hunk 1]
```diff
@@ -338,6 +340,21 @@ PDOX:
 		Duration: 20
 		KillCargo: yes
 		DisplayRadarPing: True
+	ChronoshiftPower@advancedchronoshift:
+		OrderName: AdvancedChronoshift
+		Prerequisites: structures.germany
```
Comment (pchote) [inline mods/ra/rules/structures.yaml]: Killing or unpacking a german player's last MCV will cause their chronoshift to revert to the regular type and reset the timer.  You probably want to use a specific race prerequisite that is granted by the player actor.

Comment (obrakmann) [inline mods/ra/rules/structures.yaml]: Providing the prerequisite through the player means that non-german factions will not be able to capture an advanced chronosphere, and that chronospheres captured by a german player will automatically upgrade to the advanced version. The same goes for the soviet factions' super powers. This behaviour is inconsistent with the french gap generator, so I think it should probably get the same treatment. A general question, though: do we not want this tech to be captureable?

Comment (pchote) [inline mods/ra/rules/structures.yaml]: Good point.  We can fix both sets of problems by having the chronosphere grant its own race-specific prerequisite, in the same way that we grant structures.faction, etc.

**Comment by ScottNZ** [comment]
Updated

**Comment by obrakmann** [comment]
Tested capturing chronos and conyards, both giving the expected results. :+1:

**Pull request merged**